README.md

```diff
@@ -1,2 +1,373 @@
+
+In this mod, we would like to have the `path` in the `common_dimensions` variable so that we can take advantage of it for the [annotations](#iac-checks--annotations).
+
+You can set this by passing an environment variable in the format `SP_VAR_<mod-variable-name>`.
+
+```yaml
+# Checkout your code
+  - name: Repository Checkout
+    uses: actions/checkout@v3
+# Setup Steampipe with required plugin(s) and configuration
+  - name: Setup Steampipe
+    uses: turbot/steampipe-action-setup@v1.4.0
+    with:
+      plugin-connections: |
+        connection "k8s" {
+          plugin = "kubernetes"
+          manifest_file_paths  = ["k8s/*.yaml"]
+          source_type = "manifest"
+        }
+# Run Steampipe Checks from K8s compliance
+  - name: Run K8s compliance on manifest files
+    uses: turbot/steampipe-action-check@v0.0.1
+    env:
+      SP_VAR_common_dimensions: '["namespace", "path", "source_type"]'
+    with:
+      mod-url: https://github.com/turbot/steampipe-mod-kubernetes-compliance
+```
+
+Alternatively, we can pass the `--var` command in the `additional-args` input.
+
+```yaml
+# Checkout your code
+  - name: Repository Checkout
+    uses: actions/checkout@v3
+# Setup Steampipe with required plugin(s) and configuration
+  - name: Setup Steampipe
+    uses: turbot/steampipe-action-setup@v1.4.0
+    with:
+      plugin-connections: |
+        connection "k8s" {
+          plugin = "kubernetes"
+          manifest_file_paths  = ["k8s/*.yaml"]
+          source_type = "manifest"
+        }
+# Run Steampipe Checks from K8s compliance
+  - name: Run K8s compliance on manifest files
+    uses: turbot/steampipe-action-check@v0.0.1
+    with:
+      mod-url: https://github.com/turbot/steampipe-mod-kubernetes-compliance
+      additional-args: "--var 'common_dimensions=[\"namespace\", \"path\", \"source_type\"]'"
+```
+
+### Comprehensive Example
+
+```yaml
+name: Kubernetes Compliance Checks
+on:
+  pull_request:
+    types: [opened, reopened]
+    branches: ["main"]
+    paths:
+      - k8s/*
+
+jobs:
+  k8s_compliance_checks:
+    runs-on: ubuntu-latest
+    steps:
+      # Checkout repository
+      - name: Repository Checkout
+        uses: actions/checkout@v3
+      # Install and configure Steampipe
+      - name: Steampipe Setup
+        uses: turbot/steampipe-action-setup@v1.4.0
+        with:
+          plugin-connections: |
+            connection "k8s" {
+              plugin = "kubernetes"
+              manifest_file_paths  = ["k8s/*.yaml"]
+              source_type = "manifest"
+            }
+      # Run Checks
+      - name: Steampipe Checks K8s Compliance
+        uses: turbot/steampipe-action-check@v0.0.1
+        env:
+          SP_VAR_common_dimensions: '["connection_name", "context_name", "namespace", "path", "source_type"]' # Include path dimension for annotations.
+        with:
+          mod-url: https://github.com/turbot/steampipe-mod-kubernetes-compliance # The url of the mod to clone.
```

Comment:
Instead of just adding the description as comments, can you please update the comments to only be included if we need to call out something specific (like we do above the env var)? For instance, for this one, I don't think we'd need a comment